lib/init.coffee

```diff
@@ -1,8 +1,54 @@
+{CompositeDisposable} = require 'atom'
+
 module.exports =
   config:
-    xmllintExecutablePath:
+    executablePath:
       type: 'string'
-      default: ''
-
+      title: 'xmllint Executable Path'
+      default: 'xmllint'
   activate: ->
-    console.log 'activate linter-xmllint'
+    @subscriptions = new CompositeDisposable
+    @subscriptions.add atom.config.observe 'linter-xmllint.xmllintExecutablePath',
+      (executablePath) =>
+        @executablePath = executablePath
+    @executablePath = '/usr/bin/xmllint'
+  deactivate: ->
+    @subscriptions.dispose()
+
+  lintFile: (filePath)->
+    helpers = require 'atom-linter'
+    params = ['--noout', filePath]
+    return helpers.exec(@executablePath, params, {stream: 'stderr'})
```

Comment:
This may or may not make a difference with this linter, but in many others specifying the `cwd` here solves a lot of issues. I'm not familiar with this linter so this may be a non-issue here.

(Searching for dependent files? Config files?)